### ISSUE TITLE
Investigate spurious messages shown on server startup

### ISSUE THREAD

**Issue by razzmatazz** [OmniSharp/omnisharp-emacs#280]
Currently we get spurious messages from `omnisharp-emacs` where it claims to receive unrecognized input from the server. 

```
omnisharp--handle-server-message error: (wrong-type-argument listp "The server sent an unknown json message. Inspect the omnisharp-server process buffer to view recent messages from the server. Set ‘omnisharp-debug’ to t and inspect the *omnisharp-debug* buffer for this error specifically."). See the OmniServer process buffer for detailed server output.
```

This happens 100% of the times I start the server and it is probably a low hanging fruit that can be investigated before the release.

**Comment by filipw** [comment]
this is likely caused by this https://github.com/OmniSharp/omnisharp-roslyn/issues/510

**Comment by razzmatazz** [comment]
@filipw thanks!

**Comment by razzmatazz** [comment]
```(wrong-type-argument listp "The server``` is fixed by #291 but the issue remains, as the server is still sending non-json input.

**Comment by razzmatazz** [comment]
One option, I think, could be to ignore/not report any messages from the server that are not parsable as JSON until the first one is read that is valid from that point in time DO report the errors..
